backend/src/GetMock.js

```diff
@@ -0,0 +1,16 @@
+const DynamoDB = require('aws-sdk/clients/dynamodb');
+const dynamoDb = new DynamoDB.DocumentClient();
+
+exports.handler = async function (event, context) {
+	console.log(event.path);
```

Comment:
Do we need these logs? If so can we fix the indentation?

Reply:
Removed